### DISCUSSION TITLE
context-free-botを消したい

### DISCUSSION THREAD

**Discussion by hakatashi** [tsg-ut/slackbot#565] [投票所]
このトピックは GitHub Discussions を用いた投票のテストです。

以下は @hideo54 による提案文です (#562 より引用)。

> 導入されて2年近いですが、頻度の割に好かれている印象がなく、正直いるかな? と思っています。
>
> cfp は偉大だと思いますが、cfb についてはない方がいいんじゃないかなあというのが個人的な意見です。皆さまの意見を聞きたいため issue にいたしました。

この提案に賛成の場合は :+1: を、反対の場合は :-1: でこのメッセージにリアクションしてください。

投票期限はこのメッセージがpostされてから1ヶ月以内とし、原則として賛成票と反対票の多い方を採用します。

補足意見がある場合はこのトピックにメッセージで返信してください。

**Comment by settyan117** [comment]
cfb、なくても困らないけどあっても邪魔になるほどの頻度でもないし、たまに見知った単語が出て:oxtu:ってなったり自明な単語をわからなかったりして面白いので、結構好きです

**Comment by settyan117** [comment]
（投票には参加したいが）検討中、みたいな選択肢があっても良さそう（補足意見を見て決めるというのも考えられるため）

**Reply by hideo54** [reply]
まあ 👀 とかつけてもらえればいいと思います

**Reply by settyan117** [reply]
:yosasou:

**Comment by azaika** [comment]
bot によるときたまのランダムな投稿は人間による発言の敷居を大きく下げると思っているので、皆の悪感情が強くないならば消すのには反対です。
sandbox は既に多くの発言があるため発言の促進剤としての cfb は不要だという意見もあるかもしれませんが、例えば早朝～午前にかけては発言が少ない傾向にあり、全く促進剤が不要というのは当たらないと考えます。

**Reply by hideo54** [reply]
個人的にはそもそも cfb の投稿に人間の発言を促進する効果があると感じていません。人によると言われればそれまでですが、実際のところ cfb により新たな発言や話題が生まれたケースはほとんどないと認識しています。

**Reply by hideo54** [reply]
また、発言が少ない時に発言を促進させる必要があるとも今のところあまり感じていないです。(一日中過疎るようになったら話は別ですが)

**Reply by azaika** [reply]
僕の意図していたのは cfb の投稿内容が発言を促進するというよりも、チャンネルに常に投稿がある事で投稿のハードルが下がる効果があるのではないかという話でした。これこそ個人の感想なので、そんな効果は感じないというのならそうなのかもしれません。

**Comment by settyan117** [comment]
「無くてもいい」と「無い方がいい」はかなり異なると思うんですが、「無い方がいい」という根拠はなんですか？
頻度に関しては特に何かの邪魔になった思いは個人的にはなく(そもそもsandbox自体様々な話題が飛び交う空間なので)、あまり納得出来ていません。
仮に頻度の問題だとしても、いきなり消すのではなく、まずは頻度を下げてみるという移行策の方が段階を踏んでいて適切に思えます。

**Reply by hideo54** [reply]
そもそも提案した僕自身、提案の理由の説明が足りないと思っているため、時間のある時に (リプライではなく) コメントしようと思っています。

個人的には、直接的に言えば、内容の無関係さが理由で邪魔だと思っていますが、この感覚は個人差があるため簡単に納得させられるものではないと思います。
ただし僕は、もし多数決で反対されたら、皆の場にとって意義があるということで納得します。

まずは頻度を下げるという段階を踏むのは、それはそれで良いとは思いますが、より適切な手段とは思いませんでした。正直なところそもそもcfbの意義を感じていなかったためです。
あまり意味の話を持ち出したくなく、もう少し賛成が多いものと予想していたこともあり、「頻度の割に」という説明を入れた結果、こうした意見を頂くことになり申し訳ないです。
実際に意見を聞いてみたところ好いている人が意外といたので、今はそれも良いかもしれないなと感じます。

**Comment by hakatashi** [comment]
本件は 賛成5:反対6 で否決されました。
